README.mediawiki

```diff
@@ -99,7 +99,18 @@ First review [[nep-1.mediawiki|NEP-1]]. Then clone the repository and add your N
 | Informational
 | Accepted
 |-
-| 
+| [https://github.com/neo-project/proposals/pull/156 23]
+| RPC Errors Proposal
+| Anna Shaleva, Roman Khimov
+| Standard
+| Accepted
```

Comment:
I don't think it should be here. It's not related to this PR. If #156 is merged, then we just rebase onto master.